rpm/containerd.spec

```diff
@@ -69,15 +69,6 @@ BuildRequires: gcc
 BuildRequires: systemd
 BuildRequires: libseccomp-devel
 
-# Should only return true if `el8` (rhel8) is NOT defined
-%if 0%{!?el8:1}
-%if 0%{?suse_version}
-BuildRequires: libbtrfs-devel
-%else
-BuildRequires: btrfs-progs-devel
-%endif
-%endif
```

Comment:
Not sure if this is correct; I think the intent here was to

- include `libbtrfs-devel` on centos / rhel 7 (which still had btrfs?), and Fedora (?)
- include `btrfs-progs-devel` on SUSE flavours (which default to using BTRFS as filesystem)

If I'm reading the current changes correctly, we're now excluding btrfs on all RPM variants (not just RHEL/CentOS > 7

Reply:
Actually the removal of all this related to btrfs is linked not to these "ifs", but to another one further down:

```
%if 1%{!?el8:1}
BUILDTAGS="${BUILDTAGS} no_btrfs"
%endif
```

This "if" expression can only expand to a "true" value, so my understanding is that right now, we always build with flag "no_btrfs", no matter which distro is being used. Assuming that btrfs was never used, I also removed all dependency on btrfs build requires packages.